client/my-sites/hosting/staging-site-card/card-content/manage-staging-site-card-content.tsx

```diff
@@ -51,21 +63,78 @@ const ActionButtons = styled.div( {
 	},
 } );
 
+const ActionButtonsJustifyBetween = styled( ActionButtons )( {
+	justifyContent: 'space-between',
+} );
+
 type CardContentProps = {
 	stagingSite: StagingSite;
 	onDeleteClick: () => void;
+	onPushClick: () => void;
 	isButtonDisabled: boolean;
 	isBusy: boolean;
 };
 
 export const ManageStagingSiteCardContent = ( {
 	stagingSite,
 	onDeleteClick,
+	onPushClick,
 	isButtonDisabled,
 	isBusy,
 }: CardContentProps ) => {
 	{
 		const translate = useTranslate();
+		const isStagingI3Enabled = isEnabled( 'yolo/staging-sites-i3' );
+
+		const ConfirmationPushChangesButton = () => {
+			return (
+				<ConfirmationModalButton
+					disabled={ isButtonDisabled }
+					onConfirm={ onPushClick }
+					modalTitle={ translate( 'Confirm pushing changes from your staging site' ) }
```

Comment:
Should it be production?
```suggestion
					modalTitle={ translate( 'Confirm pushing changes from your production site' ) }
```

Reply:
Its `to`, not `from`. It's `Push to Staging`. I'll fix that!